aem-cif-product-page-extension/aem/content/src/main/content/jcr_root/conf/cif-extensions/settings/dam/cfm/models/pdp-model/.content.xml

```diff
@@ -25,18 +31,18 @@
                 jcr:primaryType="nt:unstructured"
                 sling:resourceType="cq/gui/components/authoring/dialog">
                 <content
-                    jcr:lastModified="{Date}2021-04-08T12:36:55.625+03:00"
+                    jcr:lastModified="{Date}2021-06-29T13:59:39.163+03:00"
                     jcr:lastModifiedBy="admin"
                     jcr:primaryType="nt:unstructured"
                     sling:resourceType="granite/ui/components/coral/foundation/fixedcolumns">
                     <items
                         jcr:primaryType="nt:unstructured"
-                        maxGeneratedOrder="24">
+                        maxGeneratedOrder="20">
                         <_x0031_617874333419
                             jcr:primaryType="nt:unstructured"
                             sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
```

Comment:
If you use the CIF pickers in the category model, you could use it in the PDP model as well.